File: EasyDownloadSessionTests/FakeClasses/Data/NSData+DataWithContentsOfFile.m
//
//  NSData+DataWithContentsOfFile.m
//  EasyDownloadSession
//
//  Created by Javier Laguna on 08/04/2016.
//  Copyright © 2016 Javier Laguna. All rights reserved.
//

#import "NSData+DataWithContentsOfFile.h"

@implementation NSData (DataWithContentsOfFile)

+ (NSData *)dataWithContentsOfFile:(NSString *)path
{
    return [path dataUsingEncoding:NSUTF8StringEncoding];
}

@end

File: EasyDownloadSessionTests/MockClasses/Data/NSData+DataWithContentsOfFile.swift
//
//  NSData+DataWithContentsOfFile.swift
//  EasyDownloadSession
//
//  Created by Javier Laguna on 13/04/2016.
//  Copyright © 2016 Javier Laguna. All rights reserved.
//

import Foundation

extension NSData {
    
    class func new_dataWithContentsOfFile(path: String) -> NSData? {
        
        return path.dataUsingEncoding(NSUTF8StringEncoding)
    }
}